ferrilata-devilline/Controllers/ApiAdminController.cs

```diff
@@ -0,0 +1,36 @@
+﻿using ferrilata_devilline.Services.Extensions;
+using Microsoft.AspNetCore.Mvc;
+using Newtonsoft.Json.Linq;
+using System.Collections.Generic;
+
+namespace ferrilata_devilline.Controllers
+{
+    [ApiController]
+    public class ApiAdminController : Controller
+    {
+        [HttpPost("api/admin/add")]
+        public IActionResult AddAdmin([FromBody]JObject data)
+        {
+            var header = Request.Headers;
```

Comment:
Checking the header is too long, either try to simplify it here or could be shorter with an extension method probably